static/js/iframe-common.js

```diff
@@ -15,9 +15,6 @@ export function generateIFrame(domain, queryParam, urlParam) {
     var paramString = window.location.search;
     paramString = paramString.substr(1, paramString.length);
 
-    // Decode ASCII forward slash to avod repeat encodings on page refreshes
-    paramString = paramString.replace("%2F", "/");
```

Comment:
Are we able to get rid of this because `window.location.search` is already encoded?

Reply:
yeah, it's already encoded pulling from browser. I believe this was added to avoid the same double encoding problem but with multilang when the verticalUrl is something like 'es/help_articles'.